app/server.go

```diff
@@ -0,0 +1,21 @@
+package app
+
+import (
```

Comment:
By convention, the imports should be separated as follows:

    import ( 
        Go core

        Local
    
        ThirdParty
    )
   

Do this in all the imports.